data/players.yaml

```
@@ -22,6 +22,7 @@
     - '1826483' # William the Conqueror
     - '2644601' # mbl2
     - '4422938' # mbl3
+  team: Aftermath
```

Would something speak against:

```suggestion
team:
  name: Aftermath
  abbreviation: aM
  url: https://aftermath.voobly.com/
  icon: https://liquipedia.net/commons/images/thumb/3/3e/Team_Aftermath.png/600px-Team_Aftermath.png

```

---

Or having extra metadata: 

```yaml
metadata:
  teams:
    - id: 0 
      name: Aftermath
      abbreviation: aM
      url: https://aftermath.voobly.com/
      icon: https://liquipedia.net/commons/images/thumb/3/3e/Team_Aftermath.png/600px-Team_Aftermath.png

---

Would still needed to be looked-up but in the same file and would give the option to save some more information for each team (something nice to put on e.g. overlays). In fact we thought of putting the team icon in the matchmaking overlay and if we do a Twitch extension also make it clickable and link to their homepage. Hence I would really like to keep some more data in there than just a team name. Because then it's missing some essential stuff, imho.